packages/oidc-js/src/utils/sign-out.ts

```diff
@@ -35,7 +35,7 @@ import { ConfigInterface, WebWorkerConfigInterface } from "../models";
  * @returns {Promise<any>} sign out request status
  */
 /* eslint-disable @typescript-eslint/no-explicit-any */
-export function sendSignOutRequest(requestParams: ConfigInterface | WebWorkerConfigInterface): Promise<any> {
+export function sendSignOutRequest(requestParams: ConfigInterface): Promise<any> {
```

Comment:
I think we have return type as  `Promise<boolean | string>` here. Would be better to check other places where `any` is used as well. So that we can remove the lint suppress comment as well.

Reply:
This issue is tracked here: https://github.com/asgardio/asgardio-js-oidc-sdk/issues/59